examples/api/test/material/slider/slider.1_test.dart

```diff
@@ -7,29 +7,23 @@ import 'package:flutter_api_samples/material/slider/slider.1.dart' as example;
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  testWidgets('Slider shows secondary track', (WidgetTester tester) async {
+  testWidgets('Slider can change value', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('Slider can change its value', (WidgetTester tester) async {
```

Just kicking CI

Reply:
Oh I can't make a commit on this. Let em see if I can trigger a rerun on the failing check

Reply:
Done :)